[2023-11-23T22:24:15.300+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-23T21:07:10.646516+00:00 [queued]>
[2023-11-23T22:24:15.308+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-23T21:07:10.646516+00:00 [queued]>
[2023-11-23T22:24:15.308+0000] {taskinstance.py:1361} INFO - Starting attempt 13 of 13
[2023-11-23T22:24:15.319+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Data_transform_and_load> on 2023-11-23 21:07:10.646516+00:00
[2023-11-23T22:24:15.323+0000] {standard_task_runner.py:57} INFO - Started process 2420 to run task
[2023-11-23T22:24:15.326+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'COVID-19', 'Data_transform_and_load', 'manual__2023-11-23T21:07:10.646516+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmpbpogda9i']
[2023-11-23T22:24:15.328+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask Data_transform_and_load
[2023-11-23T22:24:15.370+0000] {task_command.py:416} INFO - Running <TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-23T21:07:10.646516+00:00 [running]> on host 89d301135b3a
[2023-11-23T22:24:15.623+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='COVID-19' AIRFLOW_CTX_TASK_ID='Data_transform_and_load' AIRFLOW_CTX_EXECUTION_DATE='2023-11-23T21:07:10.646516+00:00' AIRFLOW_CTX_TRY_NUMBER='13' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-23T21:07:10.646516+00:00'
[2023-11-23T22:24:16.043+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T22:24:16.160+0000] {python.py:194} INFO - Done. Returned value was: Data inserted successfully
[2023-11-23T22:24:16.176+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=COVID-19, task_id=Data_transform_and_load, execution_date=20231123T210710, start_date=20231123T222415, end_date=20231123T222416
[2023-11-23T22:24:16.221+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-23T22:24:16.232+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
